models/engine/file_storage.py

```diff
@@ -68,3 +68,18 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        all_objs = self.all(cls)
+        obj_id = cls.__name__ + '.' + id
+    
+        if obj_id in all_objs:
+            return all_objs[obj_id]
+        else:
+            return None
+    
+    def count(self, cls=None):
+        if cls != None:
+           return len(self.all(cls))
+        else:
+           return len(self.all())
```

Comment:
Hi fabian, Don't forget to add comments for each function with description about its functionality and check the pep8 standard. @Fabian-Andres